tests/simple.cpp

```diff
@@ -94,11 +94,13 @@ TEST_CASE_METHOD(SimpleFixture, "Send/Recv Test", "[simple]")
 
   // atleast one message needs to be >2GB to test for the case of multi-piece
   // messages
-  std::size_t sizes[11] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
+  static const std::size_t sizes[] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
```

Comment:
You need to explicitly specify the size in std::array, which will have to be changed whenever the list is changed. This way you can add or remove entries from the list and the size is computed by the compiler. This just seemed more convenient to me.